packages/leafygreen-provider/src/LeafyGreenProvider.tsx

```diff
@@ -1,13 +1,24 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import UsingKeyboardProvider from './UsingKeyboardProvider';
+import BaseFontSizeProvider from './BaseFontSizeProvider';
 
 interface LeafyGreenProviderProps {
   children: React.ReactNode;
+  baseFontSize?: number;
 }
 
-function LeafyGreenProvider({ children }: LeafyGreenProviderProps) {
-  return <UsingKeyboardProvider>{children}</UsingKeyboardProvider>;
+function LeafyGreenProvider({
+  children,
+  baseFontSize = 14,
```

Comment:
Perhaps the default should be defined in `BaseFontSizeProvider`.

Reply:
I think the idea here is that BaseFontSizeProvider would be included in the overall LeafyGreen provider so consumers don't need to continually add providers as we add functionality.

Reply:
👍